data.py

```diff
@@ -0,0 +1,16 @@
+import csv
+from questions import Questions
+
+
+class Source:
+
+    @staticmethod
+    def from_csv(path: str = './data.csv') -> [Questions]:
+        questions = []
+        with open(path, 'r', encoding='utf-8') as f:
+            reader = csv.reader(f)
+            # headers
+            _ = next(reader, None)
```

Comment:
What you ignored?

Reply:
header ...

Reply:
The problem is, as a reviewer, I don't know what does that, the better way at here is leaving:
next(reader, None)
only.
Else no one knows why there's a comment says headers but no headers in code

Reply:
you're right.